app/views/fae/application/_file_uploader.html.slim

```diff
@@ -28,7 +28,10 @@ ruby:
         .asset-actions.-files
           a.asset-title href=the_file.asset.url target="_blank" = the_file.asset.file.filename
 
-          = link_to '', fae.delete_file_path(the_file.id), class: 'asset-delete js-asset-delete', remote: true, method: :delete, data: { confirm: t('fae.delete_confirmation') }
+          - if the_file.id.present?
+            = link_to '', fae.delete_file_path(the_file.id), class: 'asset-delete js-asset-delete', remote: true, method: :delete, data: { confirm: t('fae.delete_confirmation') }
+          - else
+            a.js-file-clear.asset-delete.js-asset-delete
```

Comment:
`js-file-clear` and `js-asset-delete` will be misleading UI because the user will think they're deleting (the JS in `_ajax.js` doesn't know better, it's just DOM manipulation). I vote to **not** show a delete button if the file didn't save.

Reply:
Holla @jamesmk

Reply:
Given you're in a form error state - Without a delete button on the form the user is either forced to refresh the page to clear out the selected file or to correct form errors and submit the currently chosen file, and then delete it and choose a different file. Those don't seem like good user experiences if you accidentally select the wrong file or change your mind mid process.